### ISSUE TITLE
question about num_aug

### ISSUE THREAD

**Issue by RenShuhuai-Andy** [tanyuqian/learning-data-manipulation#6]
Hi, I think here in augmentation_main.py
https://github.com/tanyuqian/learning-data-manipulation/blob/315313d22d2bbadc1c6292f6f64b8b082dafd46f/augmentation_main.py#L99-L100
 `num_aug` should be set to `args.n_aug` instead of 1. Is it a typo? Or you only generate `args.n_aug` augmented texts for fine-tuning generator? But why?

**Comment by tanyuqian** [comment]
No, it's not a typo. We augment ```args.num_aug``` examples to finetune the generator (Line97) and augment ```1``` example to train the classifier (Line100 & Line102). They are two separated hyperparameters.
